Update app-scripts.md (#82)
hackforla/tables: dev-docs/app-scripts.md

- The function will send the calendar invite regardless of the timeout, but it will not run any subsequent actions if any (which is why this function was separated from the others and has its own execution for each CoP)
- By separating this automation from the others, it will allow the addUserToDrive and addUserToGithub to run in one call

### AuditGoogleDrivesPermission

> - To Run on Appscript Run the funtion "Run"

Reviewer: ```suggestion
- To Run on Appscript execute the funtion "Run"
```

> - The script go over the all the drives listed and generate each users' permission role in to a CSV file named 'google_drive_permissions_audit.csv'

Reviewer: ```suggestion
- The script runs though all the drives listed and generates a CSV (created date + 'google_drive_permissions_audit.csv') with each users' permission details 
```

- To Audit more/less drives, update driveIDs list in the Run function
- tables@hackforla.org would need permission for auditting the drive

> - The script log driveID that does not have permission to open in execution log

Reviewer: ```suggestion
- If tables@hackforla.org does not have permission to access the drive, the drive ID will appear in the execution log with the text "list permission to Drive with driveID: "+driveID+ " fails " (where +driveID+ is replaced with the drive ID
```

- The script is located in tables@hackforla.org google drive > Scripts > Google Drive Permissions > AuditGoogleDrivesPermissionAuditGoogleDrivesPermission

### GoogleformTimeTrigger

- Generate time trigger according to onboarding meeting schdule from HFLA Volunteers-[Onboarding - Schedule] table

> - The time trigger turn google form AcceptingResponses feature on/off according to on and off time from the table

Reviewer: ```suggestion
- The time trigger turns google form AcceptingResponses feature on/off according to on and off time from the table
```

> - The script weed out past events and only create triggers for 3( google has limit of 20 trigger per script) closest future events

Reviewer: ```suggestion
- The script weeds out past events and only creates triggers for the 3 closest future events.  Google limits to no more than 20 triggers per script
```

- There are 3 bots that will call the script in the [Onboarding - Schedule] table

>   - New edit trigger : call the script every time the event time is editted

Reviewer: ```suggestion
  - New edit trigger: calls the script every time the event time is editted
```

>   - weekly update trigger : call the script every week (Monday 1AM) to update the trigger

Reviewer: ```suggestion
  - weekly update trigger: calls the script every week (Monday at 1AM) to update the trigger
```

>   - weekly clear trigger : clear all triggers before weekly update

Reviewer: ```suggestion
  - weekly clear trigger: clears all triggers before weekly update, in order to prevent exceeding the 20 trigger limit
```

- To Deploy: 
  - Change formID in function openForm() & closeForm() to target google form ID on line 80 & 87
- The script is located in tables@hackforla.org google drive > Scripts > GoogleformTimeTrigger

### GoogleformTimeTrigger_Copy-for_2_form

> - same as GoogleformTimeTrigger beside turn 2 form AcceptingResponses feature on/off

Reviewer: ```suggestion
- same functions as GoogleformTimeTrigger and also allows for a second form AcceptingResponses feature on/off
```
